features/step_definitions/navigation.rb

```diff
@@ -3,6 +3,11 @@
   touch("* {text CONTAINS[c] '#{entry}'}")
 end
 
+Then (/^I hide the keyboard$/) do
+  hide_soft_keyboard()
+end
```

Comment:
This is defined here: https://github.com/dimagi/commcare-ui-tests/blob/master/features/step_definitions/main_structure_steps.rb#L23-L25